test/example_data.dart

```diff
@@ -465,13 +465,17 @@ UpdateMessageEvent updateMessageEditEvent(
 
 UpdateMessageEvent updateMessageMoveEvent(
   List<Message> messages, {
+  int? origStreamId,
   int? newStreamId,
   String? origTopic,
   String? newTopic,
   String? origContent,
   String? newContent,
 }) {
   assert(messages.isNotEmpty);
+  assert(origTopic != null, 'origTopic required for a message move');
+  assert(newTopic != null, 'newTopic required for a message move');
```

Comment:
Relatedly, if we want these asserts then they should probably just be expressed in the function's signature.

This function could also use some documentation to say how `messages` relates to the other parameters. In `updateMessageEditEvent` the Message parameter is meant to be the original, pre-edit version of the message, as reflected in its name `origMessage`. Here there's an echo of that (it's used as the default for `origStreamId`) but it doesn't seem to be applied consistently. It'd be good to have a consistent story one way or another — for example maybe it's more convenient to pass a list of the new versions of the messages, but then the function should apply that story throughout.

Reply:
> but then the function should apply that story throughout

Agreed. Will make it so that the helper always assume `messages` are the post-move messages, and express that through documentation and the removal of `newTopic` and `newStream` parameters that can then be consistently derived.